xbmc/peripherals/devices/PeripheralCecAdapter.cpp

```diff
@@ -599,6 +599,8 @@ void CPeripheralCecAdapter::OnTvStandby(void)
   case LOCALISED_ID_STOP:
     KODI::MESSAGING::CApplicationMessenger::GetInstance().PostMsg(TMSG_MEDIA_STOP);
     break;
+  case LOCALISED_ID_IGNORE:
+  	break;
```

Comment:
Looks like you accidentally used a tab for indentation.
```suggestion
    break;
```

Reply:
All OK now?